cesm/nuopc_cap_share/shr_lightning_coupling_mod.F90

```diff
@@ -0,0 +1,104 @@
+module shr_lightning_coupling_mod
+
+  !========================================================================
+  ! Module for handling namelist variables related to lightning coupling
+  !========================================================================
+
+  use ESMF         , only : ESMF_VMGetCurrent, ESMF_VM, ESMF_VMGet
+  use ESMF         , only : ESMF_LogFoundError, ESMF_LOGERR_PASSTHRU, ESMF_SUCCESS
+  use shr_sys_mod  , only : shr_sys_abort
+  use shr_log_mod  , only : shr_log_getLogUnit
+  use shr_nl_mod   , only : shr_nl_find_group_name
+  use shr_mpi_mod  , only : shr_mpi_bcast
```

Comment:
We are deprecating shr_mpi_mod, please use ESMF_VMBroadcast instead.

Reply:
fixed